modules/integration/tests-integration/bpmn/src/test/java/org/wso2/bps/integration/tests/bpmn/JavaServiceTaskDeploymentUndeploymentTestCase.java

```diff
@@ -0,0 +1,94 @@
+package org.wso2.bps.integration.tests.bpmn;
```

Comment:
Add license headers to each of the classes. Check whether license header addition is properly configured using the ide.